packages/next/src/build/webpack/loaders/next-edge-function-loader.ts

```diff
@@ -35,8 +35,7 @@ const nextEdgeFunctionLoader: webpack.LoaderDefinitionFunction<EdgeFunctionLoade
 
         enhanceGlobals()
 
-        const mod = require(${stringifiedPagePath})
-        const handler = mod.middleware || mod.default
+        import handler from ${stringifiedPagePath}
```

Comment:
Does this need to handle `export const middleware = () {}` (a `middleware` named export)? The old code seems to imply this is supported.

Reply:
edge function doesn't need `middleware` export, the only case having `middleware` export is edge middleware. this can be removed